packages/circuit-ui/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -67,46 +66,31 @@ const alignmentStyles = ({ align = 'center' }: WrapperProps) => css`
 
 const Wrapper = styled('div')<WrapperProps>(wrapperStyles, alignmentStyles);
 
-const getCurrentVariant = (theme: Theme) =>
-  window.matchMedia(theme.breakpoints.kilo).matches ? 'secondary' : 'tertiary';
+const secondaryButtonStyles = ({
+  theme,
+  destructive,
+}: ButtonProps & StyleProps) => css`
+  margin-right: ${theme.spacings.mega};
+  ${secondaryStyles({ theme, variant: 'secondary', destructive })}
+  ${theme.mq.untilKilo} {
+    margin-right: 0;
+    margin-top: ${theme.spacings.mega};
```

Comment:
Any reason why you prefer `margins` instead of `gap`?

Reply:
Also I wonder if `margin-right` would style be the correct style if we had a language going in the other direction. 🤔

Reply:
Very good question 🙂 Unfortunately we can't use flexbox `gap` yet ([caniuse for gap](https://caniuse.com/flexbox-gap) vs [our support policy](https://circuit.sumup.com/?path=/docs/introduction-browser-support--page)). `margin-block-start|end` would ideally be better but (1) it has the same browser support issues and (2) we don't support any RTL yet (by the time we do, we'll hopefully have dropped Safari 11 and we can start using start or end for spacing)